gh-actions/common/build-debian/action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
     required: false
     # FIXME: this should default to '', but we don't want to break job depending on us for now
     default: 'ca-certificates git'
+  lintian:
+    required: false
+    description: Arguments to pass to lintian, if any. Set to false to skip the lintian check.
```

Comment:
and why not `skip`?

Reply:
ah, `skip` that's fine too, I think I was basing this choice on some older PR that was suggested